### PR TITLE
{tools}[GCCcore/11.3.0] q2-krona v20220124

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by smoors** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/q/q2-krona/q2-krona-20220124-GCCcore-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+description = """QIIME2 plugin for creating Krona plots"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+builddependencies = [
+    ('binutils', '2.38'),
+]
+
+dependencies = [
+    ('Python', '3.10.4'),
```
Comment (casparvl) [inline easybuild/easyconfigs/q/q2-krona/q2-krona-20220124-GCCcore-11.3.0.eb:17]: I'm confused. If this is a QIIME2 plugin, doesn't it depend on QIIME2?

**Comment by casparvl** [comment]
Test report by @casparvl
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
tcn1.local.snellius.surf.nl - Linux RHEL 8.6, x86_64, AMD EPYC 7H12 64-Core Processor, Python 3.6.8
See https://gist.github.com/casparvl/a90f73a51a094d100358591f90eaf86b for a full test report.

File: easybuild/easyconfigs/q/q2-krona/q2-krona-20220124-GCCcore-11.3.0.eb (hunk 1 above)
Comment (casparvl) [reply easybuild/easyconfigs/q/q2-krona/q2-krona-20220124-GCCcore-11.3.0.eb:17]: By the way, feel free to ping another maintainer on Slack after you responded to this - I'm going on holiday for a while :)

As long as this was an ommission that was done on purpose, the rest of the PR looks good te me and can be merged.

Comment (smoors) [reply easybuild/easyconfigs/q/q2-krona/q2-krona-20220124-GCCcore-11.3.0.eb:17]: this was on purpose indeed.
the plugin does not depend on a specific version, so can be used with different QIIME2 versions, for example this PR has another version in the same toolchain: https://github.com/easybuilders/easybuild-easyconfigs/pull/19161

**Review by akesandgren — APPROVED** [review]
LGTM

**Comment by akesandgren** [comment]
Going in, thanks @smoors!

**Pull request merged**